pkg/modutils/mirror.go

```diff
@@ -0,0 +1,24 @@
+package modutils
+
+import (
+	"time"
+)
+
+type Module struct {
```

Comment:
this seems to only be used in tests.
can we add a test build tag to guarantee that?